src/VariableInput/VariableInput.st.css

```
@@ -1,14 +1,15 @@
 
 :import {
     -st-from: "../Foundation/stylable/colors.st.css";
-    -st-named: D80, B30, B10, F00, B50
+    -st-named: D80, B30, B10, F00, B50, D10-30
```

This import is missing:
```suggestion
    -st-named: D80, B30, B10, F00, B50, D10-30, D60
```